controllers/front/ProductController.php

```diff
@@ -954,6 +960,30 @@ protected function formatQuantityDiscounts($specific_prices, $price, $tax_rate,
         return $specific_prices;
     }
 
+    /**
+     * @param float $specificPriceValue
+     * @param bool $specificPriceReductionTax
+     * @param float $taxRate
+     * @param float $ecoTaxAmount
+     *
+     * @return mixed
+     */
+    private function calculateSpecificPrice(
+        $specificPriceValue,
+        $specificPriceReductionTax,
+        $taxRate,
+        $ecoTaxAmount
+    ) {
+        $priceDefaultCurrency = $specificPriceValue;
```

Comment:
You forget to change the variable name :trollface:

Reply:
👍 :trollface: